MapboxNavigation/RouteMapViewController.swift

```diff
@@ -268,6 +270,56 @@ extension RouteMapViewController: NavigationMapViewDelegate {
             return defaultReturn
         }
         
+        if let userLocation = mapView.userLocation,
+            let style = mapView.style,
+            routeController.showCurrentWayNameLabel == true {
+            
+            let validSources = style.sources.filter { (source: MGLSource) -> Bool in
+                if let vectorSource = source as? MGLVectorSource {
+                    if vectorSource.configurationURL?.absoluteString.range(of: "mapbox.mapbox-streets-v7") != nil {
+                        return true
+                    } else {
+                        return false
+                    }
+                } else {
+                    return false
```

Comment:
I hope we never release a `mapbox.mapbox-streets-v7.1`. 😉

Implement this extension:

```swift
extension MGLVectorSource {
    var isMapboxStreets: Bool {
        guard let configurationURL = configurationURL else {
            return false
        }
        return configurationURL.scheme = "mapbox" && configurationURL.host.components(separatedBy: ",").contains("mapbox.mapbox-streets-v7")
    }
}
```

then filter on it:

```swift
let streetsSourceIdentifiers = style.sources.flatMap {
    $0 as? MGLVectorSource
}.filter {
    $0.isMapboxStreets
}.map {
    $0.identifier
}
```